app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -142,40 +135,50 @@ public void createDrawer() {
 
         toggle.syncState();
         navigationView.setNavigationItemSelectedListener(this::onNavigationItemSelected);
-        navigationView.setCheckedItem(R.id.nav_start_activity);
+        navigationView.setCheckedItem(R.id.nav_add_activity);
     }
 
     public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
```

Comment:
All this navigation part will be replaced with the new Kepler Navigation Class. I did it in my branch 223

Reply:
Alright yes!